backend/src/ChallengeBroadcast.ts

```diff
@@ -49,7 +49,7 @@ function emitMessageSubscriber(
   if (msgParams) subscribedSocket.emit(msgTitle, msgParams);
   else subscribedSocket.emit(msgTitle);
   logMessage({
-    eloId: "SUBSCRIBER",
+    idToken: "SUBSCRIBER", // what is this for?
```

Comment:
Can you change the current comment with a question to one that answers the question?

Reply:
Meant to remove that comment lol but will do

Reply:
done